src/components/Perils/PerilModal/PerilModal.tsx

```diff
@@ -0,0 +1,269 @@
+    width: 50%;
+  }
+
+  ${TABLET_BP_UP} {
+    width: 100%;
+  }
+`
+
+const CoverageListTitle = styled.div`
+  font-size: 1.5rem;
+  color: ${colorsV3.gray900};
+  margin-bottom: 0.75rem;
+  padding-bottom: 0.75rem;
+  border-bottom: 1px solid ${colorsV3.gray900};
+`
+
+const CoverageListItem = styled.div`
+  font-size: 0.875rem;
+  line-height: 1.5;
+  color: ${colorsV3.gray900};
+  padding-left: 2rem;
+  position: relative;
+  margin-bottom: 1rem;
+
+  svg {
+    position: absolute;
+    left: 0;
+    top: 0.25rem;
+  }
+`
+
+export const PerilModal: React.FC<PerilModalProps & ModalProps> = (props) => {
+  const [actionsAllowed, setActionsAllowed] = React.useState(true)
+
+  React.useEffect(() => {
+    const isBelowBoundary = props.currentPerilIndex < props.perils.length
+    const isAboveBoundary = props.currentPerilIndex > props.perils.length * 2
+
+    if (isBelowBoundary || isAboveBoundary) {
+      setTimeout(() => {
+        props.setCurrentPeril(
+          props.currentPerilIndex +
+            (isBelowBoundary ? 1 : -1) * props.perils.length,
+        )
+      }, TRANSITION_MS)
+    }
+
+    setActionsAllowed(false)
+
+    setTimeout(() => {
+      setActionsAllowed(true)
+    }, TRANSITION_MS * 2)
+  }, [props.currentPerilIndex])
+
+  const currentPeril =
+    props.perils[props.currentPerilIndex % props.perils.length]
+
+  return (
+    <Modal isVisible={props.isVisible} onClose={props.onClose}>
+      <ModalWrapper>
+        <Header>
+          <DirectionButton
+            onClick={() =>
+              actionsAllowed &&
+              props.setCurrentPeril(props.currentPerilIndex - 1)
+            }
+          >
+            <Chevron direction="left" />
+          </DirectionButton>
+          <Title>{currentPeril.title}</Title>
+
+          <DirectionButton
+            onClick={() =>
+              actionsAllowed &&
+              props.setCurrentPeril(props.currentPerilIndex + 1)
+            }
+          >
+            <Chevron direction="right" />
+          </DirectionButton>
+        </Header>
+        <Content>
+          <Column>
+            <Description>{currentPeril.description}</Description>
+            {currentPeril.info && (
+              <InfoBox>
+                <InfoBoxTitle>{'PLACEHOLDER'}</InfoBoxTitle>
+                <InfoBoxBody>{currentPeril.info}</InfoBoxBody>
+              </InfoBox>
+            )}
+          </Column>
+          <Column>
+            <CoverageWrapper>
+              <CoverageList>
+                <CoverageListTitle>{'PLACEHOLDER'}</CoverageListTitle>
+                {currentPeril.covered.map((text) => (
+                  <CoverageListItem key={text}>
+                    <Cross size="0.75rem" />
+                    {text}
+                  </CoverageListItem>
+                ))}
+              </CoverageList>
+
+              {currentPeril.exceptions.length > 0 && (
+                <CoverageList>
+                  <CoverageListTitle>{'PLACEHOLDER'}</CoverageListTitle>
```

Comment:
is it really supposed to say `placeholder`?

Reply:
Haha nope, I was just for me not to forget to add a global field for this in storyblok 😁